### PR TITLE
List DSTRA (DST)

### PR DESCRIPTION
**Pull request by krestien**
- Official project URL: https://dstra.io
- Official block explorer URL: https://explorer.dstra.io
- Official fallback explorer URL: https://blocks.dstra.io

### CODE REVIEW

File: assets/src/test/java/bisq/asset/coins/DSTRATest.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+    @Test
+    public void testValidAddresses() {
+        assertValidAddress("DGiwGS8n3tJZuKxUdWF6MyTYvv6xgDcyd7");
+        assertValidAddress("DQcAKx5bFoeRwAEHE4EHQykyq8u2M1pwFa");
+    }
+
+    @Test
+    public void testInvalidAddresses() {
+        assertInvalidAddress("DGiwGS8n3tJZuKxUdWF6MyTYvv6xgDcyd77");
```
Comment (blabno) [inline assets/src/test/java/bisq/asset/coins/DSTRATest.java:38]: Please format the code according to Bisq code style.

File: assets/src/test/java/bisq/asset/coins/DSTRATest.java [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        assertInvalidAddress("DGiwGS8n3tJZuKxUdWF6MyTYvv6xgDcyd");
+		assertInvalidAddress("dGiwGS8n3tJZuKxUdWF6MyTYvv6xgDcyd7");
+		assertInvalidAddress("FGiwGS8n3tJZuKxUdWF6MyTYvv6xgDcyd7");
+		assertInvalidAddress("fGiwGS8n3tJZuKxUdWF6MyTYvv6xgDcyd7");
+		
+    }
+}
```
Comment (blabno) [inline assets/src/test/java/bisq/asset/coins/DSTRATest.java]: Add newline at the end of file.

**Review by blabno — CHANGES_REQUESTED** [review]
NACK per comments.

Also please squash those commits into a single one.

File: assets/src/test/java/bisq/asset/coins/DSTRATest.java (hunk 1 above)
Comment (krestien) [reply assets/src/test/java/bisq/asset/coins/DSTRATest.java:38]: Hi @blabno and thank you for checking my PR! 
After review I simply can't find the spot where I violated Bisq code style. I took a look at several examples (PIVXTest, ZcashTest, etc.) and it seems to me like my file is exactly looking the same.
Maybe you can guide me to what's wrong with my Test class. Only style reference for bisq I found is https://github.com/bisq-network/style and it doesn't contain anything regarding code style as far as I could see as it's pretty new.

**Comment by krestien** [comment]
> NACK per comments.
> 
> Also please squash those commits into a single one.

Resolved what I was able to and squashed commits, hoping for an ACK

File: assets/src/test/java/bisq/asset/coins/DSTRATest.java (hunk 1 above)
Comment (blabno) [reply assets/src/test/java/bisq/asset/coins/DSTRATest.java:38]: Lines 38-40 are not aligned.

Comment (krestien) [reply assets/src/test/java/bisq/asset/coins/DSTRATest.java:38]: Tried to fix it with IDEA (did the code with Notepad++ before) and I hope it's done now @blabno .
I can't see other alignment issues.
Squashed again, hoping for an ACK this time

**Review by blabno — APPROVED** [review]
ACK

**Pull request merged**